Makefile

```diff
@@ -210,43 +218,70 @@ deploy-tektoncd-dashboard: tektoncd-dashboard-helm
 add-bitnami-repo:
 	@helm repo add bitnami https://charts.bitnami.com/bitnami
 
-dev-dracon: deploy-elasticoperator deploy-arangodb-crds add-bitnami-repo
+deploy-cluster:
+	@scripts/kind-with-registry.sh
+
+install: deploy-cluster deploy-elasticoperator deploy-arangodb-crds add-bitnami-repo
 	@echo "fetching dependencies if needed"
 	@helm dependency build ./deploy/dracon/chart
-	@echo "deploying dracon in dev mode"
+	@echo "deploying dracon"
 	@helm upgrade dracon ./deploy/dracon/chart \
+	 	  --install \
+		  --values ./deploy/dracon/values/dev.yaml \
+		  --create-namespace \
+		  --set "image.registry=$(CONTAINER_REPO)" \
+		  --namespace $(DRACON_NS) \
+		  --version $(DRACON_VERSION) \
+		  --wait
+	@helm upgrade deduplication-db-migrations ./deploy/deduplication-db-migrations/chart \
+		  --install \
+		  --values ./deploy/deduplication-db-migrations/values/dev.yaml \
+		  --create-namespace \
+		  --set "image.registry=$(CONTAINER_REPO)" \
+		  --namespace $(DRACON_NS) \
+		  --set "image.tag=$(DRACON_VERSION)" \
+		  --wait
+
+	# we are seeting the container repo on itself is so we can override the container repo and package url from other make targets
+	$(MAKE) install-oss-components CONTAINER_REPO=$(CONTAINER_REPO) DRACON_OSS_COMPONENTS_PACKAGE_URL=$(DRACON_OSS_COMPONENTS_PACKAGE_URL)
+
+install-oss-components:
+	@helm upgrade $(DRACON_OSS_COMPONENTS_NAME) \
+		$(DRACON_OSS_COMPONENTS_PACKAGE_URL) \
 		--install \
-		--values ./deploy/dracon/values/dev.yaml \
 		--create-namespace \
 		--namespace $(DRACON_NS) \
-		--set "deduplication-db-migrations.image.tag=$(DRACON_VERSION)" \
-		--wait
-	@helm upgrade $(DRACON_OSS_COMPONENTS_NAME) oci://ghcr.io/ocurity/dracon/charts/$(DRACON_OSS_COMPONENTS_NAME) \
-		--install \
-		--namespace $(DRACON_NS) \
-		--version $$(echo "${DRACON_VERSION}" | sed 's/^v//')
-
-dev-infra: deploy-nginx deploy-tektoncd-pipeline deploy-tektoncd-dashboard
-
-dev-deploy: dev-infra dev-dracon
-
-dev-teardown:
-	@kind delete clusters dracon-demo
+		--set container_registry=$(CONTAINER_REPO)\
+		--values ./deploy/deduplication-db-migrations/values/dev.yaml
+	@echo "Done! Bumped version to $(DRACON_VERSION)"
 
 dev-update-oss-components: cmd/draconctl/bin
 	@echo "Updating open-source components in local dracon instance..."
-	@make publish-component-containers CONTAINER_REPO=localhost:5000/ocurity/dracon
+	$(eval CONTAINER_REPO:=localhost:5000/ocurity/dracon)
+
+	$(MAKE) -j 16 publish-component-containers CONTAINER_REPO=$(CONTAINER_REPO)
 	@./bin/cmd/draconctl components package \
 		--version $(DRACON_VERSION) \
 		--chart-version $(DRACON_VERSION) \
 		--name $(DRACON_OSS_COMPONENTS_NAME) \
 		./components
-	@helm upgrade $(DRACON_OSS_COMPONENTS_NAME) \
```

Comment:
why are we removing this? Previously I could just run `make dev-update-oss-components` to push the latest dev version into the cluster

Reply:
adding a new shortcut for `dev-deploy-oss-components``

Reply:
ah, I see what you're doing. Thanks for explaining!

I think in that case I'd slightly tweak the target names, to make it more clear what they do:
- `dev-update-oss-components` -> `dev-build-oss-components` maybe?

Also, can we add a convenience method that does the same as before, i.e.
```Makefile
dev-update-oss-components: build-oss-components install-oss-components
```
Otherwise we're making our life harder, from before running one command to now two.

Reply:
convenience method is `dev-deploy-oss-components`` , i'll change the naming on the other one as well